proxy/src/proxy.rs

```diff
@@ -129,6 +129,15 @@ pub static RATE_LIMITER_LIMIT: Lazy<IntGaugeVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static NUM_CONNECTION_BY_SNI: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
This is all incoming connections, not only accepted / opened connections right?
Can you please add a comment about this?

nit: it looks like more consistent naming will be `NUM_CONNECTIONS_BY_SNI_COUNTER`